Examples/Tests/particle_data_python/inputs_test_2d_particle_attr_access_picmi.py

```diff
@@ -132,7 +146,7 @@ def add_particles():
         uz=uz,
         w=w,
         newPid=newPid,
-        unique_particles=args.unique,
+        unique_particles=True,
```

Comment:
Did you intentionally remove `args.unique` here?
```suggestion
        unique_particles=args.unique,
```

It is still used in the `parser` above and potentially passed by the test runtime arguments.

Reply:
I intentionally removed it so that all added particles are treated as unique, which simplifies the subsequent check on both cpu/gpu:
`assert electrons.size == total_expected_electrons_size`,
where 
`total_expected_electrons_size = sum(
    nps_per_rank(rank) * (max_steps - 1) for rank in range(num_ranks)
)`.